src/Maestro/Maestro.Web/Api/v2019_01_10/Controllers/BuildsController.cs

```diff
@@ -0,0 +1,177 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Maestro.Data;
+using Maestro.Web.Api.v2019_01_10.Models;
+using Microsoft.AspNetCore.ApiPagination;
+using Microsoft.AspNetCore.ApiVersioning;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Swashbuckle.AspNetCore.Annotations;
+using Build = Maestro.Data.Models.Build;
+
+namespace Maestro.Web.Api.v2019_01_10.Controllers
+{
+    [Route("builds")]
+    [ApiVersion("2019-01-10")]
+    public class BuildsController : Controller
```

Comment:
These controllers seem to be a duplication of the entire class from the previous version. Why redefine everything?  There's a lot of duplicates code/logic here that we shouldn't be duplicating